### PR TITLE
chore: Remove suggestion form from settings and README

### LINKED ISSUE #2280: Remove improvement form from settings and readme
**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by rajput2122**
Fixes #2280 

Changes: Removed suggestion form from setting and readme.md

Screenshots for the change:
Screenshot_20190827-170543_Eventyay Attendee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,6 @@ If you are a tester and want to test the app, you have two ways to do that:
 
 ## Open Event Attendee Android Suggestions
 
-- Suggestion form link: [Form](https://docs.google.com/forms/d/e/1FAIpQLSd7Y1T1xoXeYaAG_b6Tu1YYK-jZssoC5ltmQbkUX0kmDZaKYw/viewform)
 - Suggestion responses link: [Sheet](https://docs.google.com/spreadsheets/d/1SzR75MBEVrTY1sDM3KAMm9wltiulDAp0QT5hv9eJkKM/edit#gid=1676755229)
```
Comment (liveHarshit) [inline README.md]: Please remove the whole section, not just the from.

Comment (rajput2122) [reply README.md]: ok sir

**Comment by rajput2122** [comment]
I have committed sir after removing suggestion sheet from README.md kindly check it

**Comment by liveHarshit** [comment]
> I have committed sir after removing suggestion sheet from README.md kindly check it

Fix Travis build

**Comment by rajput2122** [comment]
Fix travis build kindly review it sir.

**Pull request merged**